### PR TITLE
feat: adding homepage for shopping cart

### PR DESCRIPTION
**Pull request by eldhoabraham94**
feat: adding homepage for shopping cart

### CODE REVIEW

File: assets/common.css [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+
+.search-bar{
+    width:40%;
+    display: flex;
+    align-items: center;
+}
+.search-bar input{
+width: 100%;
+    padding: 0.5rem;
```
Comment (swethaogeti) [inline assets/common.css:61]: just a suggestion anything less than 1rem try using px

File: index.html [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+                        <h2 class="category-title">Baby Care</h2>
+                    </a>
+                </div>
+
+
+                <div class="img img-lg">
+                    <a href="/Product/Products.html" class="btn-primary">
+                        <img src="/assets/images/home_categoreis/toys.jpg" />
+                        <h2 class="category-title">Toys</h2>
+                    </a>
```
Comment (swethaogeti) [inline index.html:79]: reset the code !

File: index.html [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+                    <a href="/Product/Products.html" class="btn-primary">
+                        <img src="/assets/images/home_categoreis/travel.jpg" />
+                        <h2 class="category-title">Travel</h2>
+                    </a>
+                </div>
+
+
+            </div>
+        </div>
+
```
Comment (swethaogeti) [inline index.html:93]: clean code ! everything looks good ! GTM

**Review by swethaogeti — APPROVED** [review]
good ! can merge it

File: assets/common.css (hunk 1 above)
Comment (eldhoabraham94) [reply assets/common.css:61]: Okay. will update

File: index.html (hunk 2 above)
Comment (eldhoabraham94) [reply index.html:79]: Sorry I didn't get.. Reset??

File: index.html (hunk 3 above)
Comment (eldhoabraham94) [reply index.html:93]: Thank you

**Review by Debarshi95 — APPROVED** [review]
Looks good. Good to merge.

**Comment by eldhoabraham94** [comment]
> Looks good. Good to merge.

thank you

**Pull request merged**